build.sbt

```diff
@@ -13,8 +13,10 @@ val sttpVersion = "3.5.2"
 val circeVersion = "0.14.1"
 val catsEffectVersion = "3.3.14"
 val fs2Version = "3.2.11"
+val cogniteAuthVersion = "2.5.3"
 
 lazy val gpgPass = Option(System.getenv("GPG_KEY_PASSWORD"))
+val artifactory = "https://cognite.jfrog.io/cognite/"
```

Comment:
feel weird that we have to include these internal cognite auth library 🙈 Any suggestion to avoid this is welcome, probably my approach is not a reasonable one ...